src/boundaries/edgetracking.jl

```diff
@@ -149,9 +176,13 @@ function edgetracking(pds::ParallelDynamicalSystem, mapper::AttractorMapper;
     
     track1 = StateSpaceSet(reduce(hcat, track1)')
     track2 = StateSpaceSet(reduce(hcat, track2)')
-    edge = StateSpaceSet((Matrix(track1) + Matrix(track2))/2)
 
-    return edge, track1, track2
+    return EdgeTrackingResults(
+        StateSpaceSet((Matrix(track1) + Matrix(track2))/2),
```

Comment:
```suggestion
        StateSpaceSet((track1 .+ track2)./2),
```
Welcome to the world of broadcasting.